pytest_mccabe.py

```diff
@@ -57,15 +63,22 @@ class McCabeError(Exception):
 
 class McCabeItem(pytest.Item, pytest.File):
 
-    def __init__(self, path, parent, complexity):
-        super(McCabeItem, self).__init__(path, parent)
+    def __init__(self, fspath, parent, complexity):
+        super(McCabeItem, self).__init__(fspath, parent)
         self._mtime = None
         if hasattr(self, 'add_marker'):
             self.add_marker("mccabe")
         else:
             self.keywords["mccabe"] = True
         self.complexity = complexity
 
+    @classmethod
+    def from_parent(cls, parent, fspath, **kwargs):
```

Comment:
Could you explain why this override is needed? I think it should be possible to pass additional arguments into `from_parent` and get them in `__init__` without anything like that, no?

Reply:
I don't remember why I thought it was necessary. I think the renaming of path to fspath was, but not the override and I struggled a bit to find out ... I removed the override :-)